src/storage/file_manager.c

```diff
@@ -5099,7 +5130,7 @@ file_alloc (THREAD_ENTRY * thread_p, const VFID * vfid, FILE_INIT_PAGE_FUNC f_in
     }
 
   assert (!VPID_ISNULL (vpid_out));
-  file_header_sanity_check (fhead);
+  file_header_sanity_check (thread_p, fhead);
```

Comment:
move file_header_sanity_check after exit label and after if (is_sysop_started) block. this is again a midpoint of file_alloc execution and header is not necessarily sane. It should be sane at the end of operation (successful or failed).